Documentation~/modes.md

```diff
@@ -9,7 +9,7 @@ You can use ProBuilder tools in two tool contexts:
     * Edge selection: Select edges and perform semi-complex geometry editing, and edge loop modeling techniques. For a complete list of actions you can perform in this mode, refer to [Edge actions](edge.md).
     * Face selection: Select faces on an object to perform basic tasks like moving, extruding, or even deleting them. For a complete list of actions you can perform in this mode, refer to [Face actions](face.md).
 
-To change context, in the **Tools** overlay, select the **Tool Context** button.
+To change context, in the **Tools** overlay, select the **Tool Context** button at the top of the overlay.
```

Comment:
We're not really supposed to direct people to where things are in the UI (or even use the word "button", which I know I did :) )

Reply:
True, but at the same time I really worry that users who aren't familiar with the pattern will struggle to find the context.

Reply:
A compromise. What do you think? 
```suggestion
To change context, in the **Tools** overlay, select the **Tool Context** at the top of the overlay.
```

Reply:
I hate it, but okay.